src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Counter App To Keep Count" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Counter App</title>
+</head>
+<body>
+    <div id="app">
+        <h1 class="app-Name">Keep Count</h1>
+        <h2 class="value"></h2>
+        <div class="buttons-div">
+            <button class="buttons" id="minus-button" type="button" >-</button>
```

Comment:
On this line, as well as lines 16 and 17, there are some extra spaces around some of the class names and after the `type` attribute. 

You could add prettier either in your editor or in your dev environment to help with consistent formatting throughout your code. It would help with removing whitespaces, enforcing consistent tabs or spacing and others. There are a number of options you can utilize. But of course, this is optional.